src/test/java/gov/loc/repository/bagit/verify/BagVerifierTest.java

```diff
@@ -118,6 +120,10 @@ public void testVersion0_97IsValid() throws Exception{
   @Test
   public void testVersion0_98IsValid() throws Exception{
     rootDir = Paths.get(getClass().getClassLoader().getResource("bags/v0_98/bag").toURI());
+    if (System.getProperty("os.name").contains("Windows")){
+    	Path bagitDir = rootDir.resolve(".bagit");
```

Comment:
John - Actually BagVerifierTest.testVersion0_98IsValid() and ReaderWriterVerifierIntegrationTest.testReaderWriterVersion98() only work successfully if the .bagit folder is marked as hidden (explicitly in Windows, it's happening implicitly in Unix) - They both invoke
BagVerifier.isValid() 
	BagVerifier.isComplete(), 
		BagVerifier.checkAllFilesInPayloadDirAreListedInAManifest() which employs PayloadFileExistsInManifestVistor, where preVisitDirectory() is bypassing .bagit directory as a payload directory,
Otherwise these tests fail because .bagit/tagmanifest-md5.txt is not listed in any manifest